codes/config.py

```
@@ -38,3 +38,32 @@ def __getitem__(self, key: str):
         # Alias to maintain old API: return initial epsilon value
         return self.epsilon_start
+    def __post_init__(self):
+        # Trust-related params
+        if not (0.0 <= self.initial_trust <= 1.0):
+            raise ValueError(f"initial_trust must be in [0.0, 1.0], got {self.initial_trust}")
+        if not (0.0 <= self.trust_decay <= 1.0):
+            raise ValueError(f"trust_decay must be in [0.0, 1.0], got {self.trust_decay}")
+        if not (0.0 <= self.trust_reward_weight <= 1.0):
+            raise ValueError(f"trust_reward_weight must be in [0.0, 1.0], got {self.trust_reward_weight}")
+        if not (0.0 <= self.trust_penalty_weight <= 1.0):
+            raise ValueError(f"trust_penalty_weight must be in [0.0, 1.0], got {self.trust_penalty_weight}")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)